nengolib/signal/lyapunov.py

```diff
@@ -337,11 +340,13 @@ def _normtail(sig, A, x, C):
         x = _state_impulse(Phi, x0=B, k=N, delay=0)  # eq (38)
         abs_e = np.squeeze(abs(C.dot(x.T)))
         x = x[:-1]  # compensate for computing where thresholds crossed
+        abs_y = abs_y[:-1]  # compensate for computing where thresholds crossed
```

Comment:
Removing this line gives a warning that I was ignoring before, due to `abs_y[cross]` having mismatched shapes between the boolean index `cross` and `abs_y` (latter had an extra dimension).